lib/view-models/vim-command-mode-input-view.coffee

```diff
@@ -24,35 +29,29 @@ class VimCommandModeInputView extends View
 
   handleEvents: ->
     if @singleChar?
-      @editor.find('input').on 'textInput', @autosubmit
-    @editor.on 'core:confirm', @confirm
-    @editor.on 'core:cancel', @cancel
-    @editor.find('input').on 'blur', @cancel
-
-  stopHandlingEvents: ->
-    if @singleChar?
-      @editor.find('input').off 'textInput', @autosubmit
-    @editor.off 'core:confirm', @confirm
-    @editor.off 'core:cancel', @cancel
-    @editor.find('input').off 'blur', @cancel
-
-  autosubmit: (event) =>
-    @editor.setText(event.originalEvent.data)
+      @editorElement.addEventListener('textInput', @autoSubmit)
```

Comment:
Yeah, I can't remember why that doesn't work. Observing text changes on the model seems to work ok though.